### DISCUSSION TITLE
Welcome to PINGMapper Discussions!

### DISCUSSION THREAD

**Discussion by CameronBodine** [CameronBodine/PINGMapper#73] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by henzb72** [comment]
Hi Dr Cameron,
Thank you for creating this discussion forum. I am Hendra from Indonesia and I am trying to install and use the Ping-mapper software to process my Humminbird sonar data. 
Cheers.

**Reply by CameronBodine** [reply]
Hi Hendra,
Thanks for being an early adopter of PING-Mapper! I look forward to hearing how it works for you. If you have any difficulties with the software, please submit an [Issue](https://github.com/CameronBodine/PINGMapper/issues).

**Comment by gushangzao007** [comment]
nice work! I'm Xi from china,  I am wondering if it is possible that I can create a 3D Modle file (like *.obj/*.stl) from Humminbird sonar data? if it is ok, any suggestion about how to do it?

**Reply by CameronBodine** [reply]
Hi Xi, I'm happy you found PINGMapper! If I understand correctly, do you want to create a bathymetric profile with Humminbird? If so, it is important to know that Humminbird side imaging systems can only measure depth at nadir, so creating a bathymetric profile would require many tightly spaced transects to generate an accurate 3D profile.

PINGMapper does export the depth measured by the instrument and exports that data to .csv. (see [this page](https://cameronbodine.github.io/PINGMapper/docs/gettingstarted/Exports.html#ping-attributes-to-csv) for more info.) I'm sure this could be converted to the files types you are interested in, but I'm unsure how.

You may be interested in looking into the Lowrance sonar system. These units are able to measure depth across the swath of the scan. Take a look at [this paper](https://www.mdpi.com/2220-9964/9/3/149). PINGMapper doesn't support files from Lowrance, but the authors did develop an [open-source tool](https://github.com/halmaia/SL3Reader) which you may find useful.

Hope this helps!

**Reply by gushangzao007** [reply]
Thank you very much! buddy!  it's very helpful for my Hobby project：）
